modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/configuration/CKEditor5Configuration.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.frontend.editor.ckeditor.web.internal.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Marko Cikos
+ */
+@ExtendedObjectClassDefinition(generateUI = false)
+@Meta.OCD(
+	id = "com.liferay.frontend.editor.ckeditor.web.internal.configuration.CKEditor5Configuration"
+)
+public interface CKEditor5Configuration {
+
+	@Meta.AD(deflt = "GPL", required = false)
```

Comment:
Default value allows anyone to do `ant all` and get a working CKEditor5 with a watermark.

This is probably fine, but I'd appreciate confirmation from @herrardo 

As discussed, we have to consider all cases:

- People building liferay portal on their own: they will have the `GPL` licensing
- People using the liferay portal bundles/docker images: they'd get the open source license
- People using the liferay DXP bundles/docker images: they'd get the commercial license
- Internal uses:
  -  Local dev machines: in most cases, GPL/dev license would be fine unless we want to test a premium feature
  - CI/PW tests: we'd like development licenses to be available most likely